corehq/apps/hqwebapp/two_factor_gateways.py

```diff
@@ -59,10 +62,18 @@ def send_sms(self, device, token):
             return HttpTooManyRequests()
 
         message = _('Your authentication token is %s') % token
-        self.client.api.account.messages.create(
-            to=device.number.as_e164,
-            from_=self.from_number,
-            body=message)
+        try:
+            self.client.api.account.messages.create(
+                to=device.number.as_e164,
+                from_=self.from_number,
+                body=message)
+        except TwilioRestException as e:
+            request = get_request()
+            if request:
+                notify_exception(request, str(e))
```

Comment:
this should be outside of the `if` - it will handle the `None` request fine.

Reply:
fixed